kafka-ui-api/src/main/java/com/provectus/kafka/ui/mapper/KafkaConnectMapper.java

```diff
@@ -41,6 +41,9 @@ default FullConnectorInfo fullConnectorInfoFromTuple(InternalConnectInfo connect
         .map(TaskStatus::getState)
         .filter(ConnectorTaskStatus.FAILED::equals)
         .count();
+    if (tasks.size() != 0 && failedTasksCount == tasks.size()) {
```

Comment:
1. I think it would be better to return failed status event 1 task failed
2. Let's create new status for it, because Connector FAILED != Tasks Failed and vice verse